Home view and layout changes

hfej opened the pull request:
- Added home view
- Changed layout colors
- Added links to external websites
- Changed navbar and footer

hfej committed:
Visual coherence changes

Changed navbar and search bar colors and shapes in order to keep the design coherent

hfej committed:
Added simple footer

hfej committed:
Added homepage layout

hfej committed:
Added image logo to navbar

hfej committed:
Added external site links

fhtoma committed:
Smart References v-for

hfej committed:
Changed image sources

fhtoma committed:
Assets Folder Reorganization

fhtoma committed:
Vuetify Colors and Better Logic with Breakpoints

fhtoma committed:
Footer Year Improvement

fhtoma reviewed (approved):
Poczytaj zmienione pliki

fhtoma committed:
Home Improvements

- router-link for future navigation
- using id from database
- small improvements

hfej committed:
Added movie title tooltip
